CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-## TBD - not ready for production
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.0.3] TBD
```

Comment:
remove [0.0.3] just leave TBD it's confusing if you see 0.0.3 it looks like it has been released already.

Reply:
ok.